### DISCUSSION TITLE
Difference between `Form` and `Fetcher` is a navigation?

### DISCUSSION THREAD

**Discussion by jmaldon1** [remix-run/remix#3882] [Q&A] [answered]
What if I simply don't redirect in the action on the submission of a `Form` ? What if I do redirect in the action on the submission of a `Fetcher.Form`?

Does the ability to do this make them equivalent?

**Comment by kiliman** [comment, marked as answer]
It depends. Form will navigate to the action route. Fetchers do not. If the action is blank or to the current route then yes it’s similar to what a fetcher does. However the data is accessed via useActionData instead of fetcher.data.
